src/plugins/dataflow-tool/components/dataflow-program.tsx

```diff
@@ -34,9 +34,10 @@ import { DataflowProgramZoom } from "./ui/dataflow-program-zoom";
 import { DataflowProgramGraph,DataSet, ProgramDisplayStates } from "./ui/dataflow-program-graph";
 // import { uploadProgram, fetchProgramData, fetchActiveRelays, deleteProgram } from "../utilities/aws";
 import { NodeChannelInfo, NodeSensorTypes, NodeGeneratorTypes, ProgramDataRates, NodeTimerInfo,
-         virtualSensorChannels } from "../model/utilities/node";
+         virtualSensorChannels, serialSensorChannels} from "../model/utilities/node";
 import { Rect, scaleRect, unionRect } from "../utilities/rect";
 import { DocumentContextReact } from "../../../components/document/document-context";
+import { SerialDevice } from "src/models/stores/serial";
```

Comment:
Should be a relative URL; there's a VSCode setting to configure auto-imports to prefer relative URLs.

Reply:
thanks, that worked